README.md

```diff
@@ -1,55 +1,81 @@
 <img alt="NL Portal Logo" src=".github/readme/images/nl-portal-logo.svg">
 
-![Version 2.0.1](https://img.shields.io/badge/Version-2.0.1-blue)  
+![Version 2.0.1](https://img.shields.io/badge/Version-2.0.1-blue)
 
 This repository contains reference implementations for the NL Portal Backend and Frontend Apps.
 It also contains a Docker Compose file for starting up a pre-configured NL Portal Demo without having to write any code.
 
+#### Compose file structure
+
+The compose file is build up out of multiple sets of containers (profiles) that together enable certain functionality
+e.g. the `haalcentraal` profile contains services for haalcentraal-brp and haalcentraal-bewoning so you can choose to
+not compose those if you don't require them. If a service is missing a profile, that means it's a core requirement
+for the app and will be composed by default.
+
+| Service                          | Port  | Profile      |
+|----------------------------------|-------|--------------|
+| Pre-built NL Portal App Backend  | 8080  | remote       |
+| Pre-built NL Portal App Frontend | 3000  | remote       |
+| DIY NL Portal App Backend        | 8080  | local        |
+| DIY NL Portal App Frontend       | 3000  | local        |
+| NL Portal Database               | 54321 | -            |
+| Keycloak                         | 8082  | -            |
+| Open Zaak                        | 8001  | zgw          |
+| Objecten API                     | 8010  | zgw          |
+| Objecttypen API                  | 8011  | zgw          |
+| Open Notificaties                | 8012  | zgw          |
+| OpenKlant                        | 8013  | zgw          |
+| OpenKlant 2                      | 8014  | zgw          |
+| Haalcentraal BRP                 | 5010  | haalcentraal |
+| Haalcentraal Bewoning            | 5011  | haalcentraal |
+
+All of the above services are set up to expose their ports via the helper server `localhost`. This service defines the
+external port mapping.
+
+**NB! The profiles `remote` and `local` can not be run at the same.** 
```

Comment:
Yay, great improvement :)
This sentence ends a bit weird; at the same time OR concurrently I guess.

Reply:
good one. the sentence was initially longer so I cut it short (and ate a word haha) :P